RealTimeFaceMaskDetector/TCPServer/SocketServer.cpp

```diff
@@ -130,6 +130,11 @@ bool SocketServer::ReceiveFullMessage()
 {
 	int total_bytes_count = GetMessageLength();
 
+	if (total_bytes_count == 0)
```

Comment:
This will not work. Sometimes it receives more than zero bytes. It could be 2,3,7 etc.